exampleSite/content/about-split/content-split.md

```diff
@@ -0,0 +1,42 @@
++++
+fragment = "content-split"
+#disabled = true
+date = "2016-09-07"
+weight = 100
+#background = ""
+title = "About Syna"
+subtitle = ""
+
+content = """
```

Comment:
Content is easier to be done within the actual content section. Makes formatting for the user easier. Basically moving this to after `++++` and then calling it via .Content should work.

Reply:
If that doesn't work, it would make the whole interface for the user a pain for content.

Reply:
Actually tried that solution and it didn't work. Entire resource scope did not contain the markdown that comes after frontmatter. No idea why that happens though. And I agree this is ridiculous but I have no idea how to ease the pain.

Reply:
Damn will look into it.